web/components/admin/TextField.tsx

```diff
@@ -128,25 +150,66 @@ export const TextField: FC<TextFieldProps> = ({
         </div>
       ) : null}
 
-      <div className="input-side">
-        <div className="input-group">
-          <Field
-            id={fieldId}
-            className={`field ${className} ${fieldId}`}
-            {...fieldProps}
-            {...(type !== TEXTFIELD_TYPE_NUMBER && { allowClear: true })}
-            placeholder={placeholder}
-            maxLength={maxLength}
-            onChange={handleChange}
-            onBlur={handleBlur}
-            onPressEnter={handlePressEnter}
-            disabled={disabled}
-            value={value as number | (readonly string[] & number)}
-          />
+      {!isAdminPwdField ? (
```

Comment:
Similar comment, since these text fields aren't password specific. If what you're splitting up are fields with complexity requirements and fields without complexity requirements maybe the bool should be called `hasComplexityRequirements`.